bookwyrm/models/book.py

```diff
@@ -293,3 +301,14 @@ def isbn_13_to_10(isbn_13):
     if checkdigit == 10:
         checkdigit = "X"
     return converted + str(checkdigit)
+
+
+@receiver(models.signals.post_save, sender=Edition)
+# pylint: disable=unused-argument
+def preview_image(instance, *args, **kwargs):
+    changed_fields = {}
+    if instance.field_tracker:
+        changed_fields = instance.field_tracker.changed()
```

Comment:
I run into a problem wih the `ShelfBook` model, so this will ensure that `changed_fields` is only defined if the sender has a `field_tracker` field. I don't know if it's the best way.